### ISSUE TITLE
When we install clusterpedia, it restarts for more than 5 times

### ISSUE THREAD

**Issue by yankay** [clusterpedia-io/clusterpedia#85]
### What happened?

![企业微信截图_16454971838462](https://user-images.githubusercontent.com/1469319/155052457-83b23d82-e405-4cf9-9b2c-518174c5f667.png)

When we install clusterpedia, it restarts for more than 5 times

### What did you expect to happen?

The Clusterpedia not restart, it wait for mysql ready.

### How can we reproduce it (as minimally and precisely as possible)?

Install Clusterpeida using Kpanda

### Anything else we need to know?

We can use initContainer to check mysql ready. Eg:

- https://stackoverflow.com/questions/65646499/how-to-use-an-init-container-to-check-if-mysql-is-ready-for-connections
- https://www.magalix.com/blog/kubernetes-patterns-the-init-container-pattern

### Clusterpedia Version

<details>

```console
# replace the pod name of clusterpedia apiserver
$ kubectl -n clusterpedia-system exec {clusterpedia-apiserver-pod-name} -- apiserver --version=raw
# paste output here

# replace the pod name of clusterpedia clustersynchro-manager
$ kubectl -n clusterpedia-system exec {clustersynchro-manager-pod-name} -- clustersynchro-manager --version=raw
# paste output here
```

</details>

### Host Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

**Comment by Iceber** [comment]
@yankay Thanks for your feedback!

Since clusterpedia is not restricted to using `mysql` or `postgres`, and it is difficult to tell in the simplest manifest whether `mysql` or `postgres` is being used.

In the future, the type of storage component will be determined in the helm and the readiness of the storage component will be checked in advance. https://github.com/clusterpedia-io/clusterpedia/pull/53

It will also be considered that the `storage layer` will provide the checks on the storage components, and then the url for health checks will reflect the readiness of the storage components.

**Comment by Iceber** [comment]
[#134 ](https://github.com/clusterpedia-io/clusterpedia/pull/134) has not been merged, and the issue does not affect the 0.3 release, ready to be fixed in 0.4

**Comment by calvin0327** [comment]
@Iceber @RuliXu  I think the [pr](https://github.com/clusterpedia-io/clusterpedia/pull/199)  is better.  close the previous pr.
